openstf/model/confidence_interval_applicator.py

```diff
@@ -41,7 +51,7 @@ def _add_standard_deviation_to_forecast(
                 "forecast", "stdev"
         """
 
-        standard_deviation = self.confidence_interval
+        standard_deviation = self.model.confidence_interval
```

Comment:
To be this is confusion. I would suggest we choose one name for one thing. Either we call is `standard_deviation` or we call it `confidence_interval`. Or do we really have a good reason to use a different name for the same thing?

Reply:
I agree this is a bit confusing. We have i fact two ways to make a confidence interval: quantiles and the stdev. I renamed everything that only makes/applies the stdev to standard_deviation(including the example above). The confidence_interval_applicator is more broader as it also adds the quantiles, I keeped that name.